### ISSUE TITLE
[Package Issue]: GIGABYTE.GigabyteControlCenter

### ISSUE THREAD

**Issue by DrillWizard** [microsoft/winget-pkgs#202058]
### Please confirm these before moving forward

- [x] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [x] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

GIGABYTE.GigabyteControlCenter.installer.yaml specifies "InstallerUrl: https://download.gigabyte.cn/FileList/Utility/GCC_24.12.06.01.zip" but after successful installation, a previous version is installed.  

### Steps to reproduce

1. `winget update GIGABYTE.GigabyteControlCenter`
2. `cd C:\Program Files\GIGABYTE\Control Center`
3. `Get-ChildItem -Recurse -File | Where-Object { $_.Extension -notin ".exe", ".dll" } | Select-String -Pattern "24.10.28.01" -CaseSensitive:$false`
4. Look at the versions listed.  Additionally, run `winget update` again and see that 24.12.06.01 is still not installed.

### Actual behavior

`winget update GIGABYTE.GigabyteControlCenter` is successful but continues to show an available update when running `winget update` again. 

### Expected behavior

update to the version listed in `winget update`

### Environment

```raw
Windows: Windows.Desktop v10.0.26120.2705
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.25.40.0
```

**Comment by rakleed** [comment]
Fixed in https://github.com/microsoft/winget-pkgs/pull/201791

**Comment by Dragon1573** [comment]
> Fixed in [#201791](https://github.com/microsoft/winget-pkgs/pull/201791)

I think the issue can be manually closed as completed.

CC @DrillWizard @stephengillie 🙏🏼

**Comment by stephengillie** [comment]
> > Fixed in [#201791](https://github.com/microsoft/winget-pkgs/pull/201791)
> 
> I think the issue can be manually closed as completed.
> 
> CC [@DrillWizard](https://github.com/DrillWizard) [@stephengillie](https://github.com/stephengillie) 🙏🏼
